src/cell_value.rs

```diff
@@ -0,0 +1,104 @@
+use std::borrow::Cow;
+
+use leptos::{view, IntoView};
+
+#[derive(Default)]
+pub struct NumberRenderOptions {
+    pub precision: Option<usize>,
+}
+// A value that can be rendered as part of a table
+pub trait CellValue {
+    type RenderOptions;
```

Comment:
Can we specify a default of `()`? Or is that only possible in nightly?

Reply:
Ah it was type bounds that was experimental, right? Because we should have `Default` as a requirement here.
If that is not possible we should add it to the docs. This type should have doc comments anyway.

Reply:
Was able to get the `Default` bound here set up, but seems like setting a default is a nightly thing. Tried to improve the docs related to this but maybe we should have an example on how to implement this trait for your own type where `RenderOptions` is the empty tuple `()`

Reply:
Awesome! There are lots of examples in the code base for now 👍🏼